src/extension/notifier/pom.xml

```diff
@@ -0,0 +1,61 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/maven-v4_0_0.xsd ">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>au.org.emii.wps</groupId>
+    <artifactId>notifier</artifactId>
+    <packaging>jar</packaging>
+
+    <name>WPS completion notifier</name>
+
+    <parent>
+        <groupId>org.geoserver</groupId>
+        <artifactId>extension</artifactId>
+        <version>1.0.0</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.geoserver.extension</groupId>
+            <artifactId>gs-wps-core</artifactId>
+            <version>${geoserver.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.1</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.11</version>
```

Comment:
This dependency is *not* specified in the parent pom(s) (which is why removing the version broke things).  But perhaps junit is used by enough component to be worthy of having its dependency in the top level pom?  The advantage is less code + every component using the same version of junit.  Disadvantage is more complicated root pom, doesn't apply to *all* components.

Thoughts @jonescc?